analyzers/its/expected/Automapper/AutoMapper--net461-S3900.json

```diff
@@ -2173,6 +2173,32 @@
 },
 {
 "id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'sourceMemberInfo' before using it.",
+"location":  {
+"uri":  "sources\Automapper\src\AutoMapper\QueryableExtensions\QueryMapperVisitor.cs",
+"region":  {
+"startLine":  224,
+"startColumn":  51,
+"endLine":  224,
+"endColumn":  67
+}
+}
+},
+{
+"id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'sourceMemberInfo' before using it.",
+"location":  {
+"uri":  "sources\Automapper\src\AutoMapper\QueryableExtensions\QueryMapperVisitor.cs",
+"region":  {
+"startLine":  231,
```

Comment:
Looks good
https://github.com/SonarSource/sonar-dotnet/blob/3420f6cc2ab44de7fba95030d2770633430ec625/analyzers/its/sources/AutoMapper/src/AutoMapper/QueryableExtensions/QueryMapperVisitor.cs#L222-L224

Reply:
I've added back the deleted line, and now it's gone. There was a PR earlier regarding Flow Capture which would have fixed it, but we decided to cut branches, so this will remain a FN.